docs/howto/environment.md

```diff
@@ -15,35 +15,41 @@
   ```
   For comparison devs have used node at 10.13.x and 12.x.x.
 
-3. Install development copies of cylc and cylc-uiserver:
+3. Install
+   [cylc/cylc-flow](https://github.com/cylc/cylc-flow/) and
+   [cylc/cylc-uiserver](https://github.com/cylc/cylc-uiserver/):
   ```bash
-  pip install -e $DEVDIR/cylc
-  pip install -e $DEVDIR/cylc-uiserver
+  pip install -e "${DEVDIR}/cylc-flow" -e "${DEVDIR}/cylc-uiserver"
   ```
-4. Install npm in cylc-ui client.
+  where `${DEVDIR}/cylc-flow` and `${DEVDIR}/cylc-uiserver` are the locations
+  of your local (git) worktrees of the two projects.
+
+4. Install [cylc/cylc-ui](https://github.com/cylc/cylc-ui/).
   ```bash
   cd $DEVDIR/cylc-ui
-  npm install
+  npm install --prefix="${DEVDIR}/cylc-ui"
   npm install -g configurable-http-proxy
```

Comment:
@kinow Do you know why we need to install `configurable-http-proxy` separately?

Reply:
If you install `jupyterhub` via `pip`, then you [need to install the proxy separately](https://jupyterhub.readthedocs.io/en/stable/quickstart.html#prerequisites).

You can install `jupyterhub` via Conda, and that installs the `configurable-http-proxy` as well (it might be useful for our packaging in Cylc now that I think about it...)

Perhaps using conda could be simpler, though I don't know if we have something like editable installations with Conda.